### PR TITLE
Create a lib/ directory

### PR DESCRIPTION
**Pull request by iskakaushik**
Fixes: https://github.com/flutter/devtools/issues/1148

### CODE REVIEW

File: packages/devtools/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 0.1.9 - 2019-10-09
```
Comment (kenzieschmoll) [inline packages/devtools/CHANGELOG.md]: Does this change need to be published right away? we have a publish planned for (hopefully) later this week, so is it fine if we wait until then?

File: packages/devtools/pubspec.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@ description: A suite of web-based performance tooling for Dart and Flutter.
 # package remaining purely as a container for the compiled copy of the devtools
 # app. That ensures that version constraints for the devtools_app do not impact
 # this package.
-version: 0.1.8
+version: 0.1.9
```
Comment (kenzieschmoll) [inline packages/devtools/pubspec.yaml]: if this change does need to be published right away, please update the version numbers in the files specified by the above code comments.

**Comment by iskakaushik** [comment]
@kenzieschmoll later in the week is fine. I'll revert the version bumps.

**Comment by iskakaushik** [comment]
Seems like the tree is currently red.

**Comment by jacob314** [comment]
lgtm

**Comment by kenzieschmoll** [comment]
If you pull in master I think the bot failure should go away.

**Pull request merged**